openwrt-openwisp-monitoring/files/lib/openwisp/dhcp.lua

```diff
@@ -1,42 +1,32 @@
-package.path=package.path .. ";../files/lib/?.lua"
+package.path = package.path .. ";../files/lib/?.lua"
 
 -- retrieve dhcp leases
-local utils=require('openwisp.monitoring_utils')
-local uci=require('uci')
-local uci_cursor=uci.cursor()
-local io=require('io')
-local dhcp={}
+local utils = require('openwisp.monitoring_utils')
+local uci = require('uci')
+local uci_cursor = uci.cursor()
+local io = require('io')
+local dhcp = {}
 
 function dhcp.parse_dhcp_lease_file(path, leases)
-  local f=io.open(path, 'r')
-  if not f then
-    return leases
-  end
+  local f = io.open(path, 'r')
+  if not f then return leases end
   for line in f:lines() do
-    local expiry, mac, ip, name, id=line:match('(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)')
-    table.insert(leases, {
-      expiry=tonumber(expiry),
-      mac=mac,
-      ip=ip,
-      client_name=name,
-      client_id=id
-    })
+    local expiry, mac, ip, name, id = line:match('(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)')
+    table.insert(leases, {expiry = tonumber(expiry), mac = mac, ip = ip, client_name = name, client_id = id})
```

Comment:
this line is quite long, is there a way to set a max length so that long lines are splited on multiple lines?

Reply:
What should be the max_length? Luacheck checks if line is greater than 120 characters or not. Hence, I followed its convention and set the max_length to 120

Reply:
80 characters is the traditional limit